### ISSUE TITLE
Faker creates invalid phone numbers

### ISSUE THREAD

**Issue by TheMcMurder** [Marak/faker.js#297]
faker will occasionally create a phone number like this (faker.phone.phoneNumberFormat(2))  
`1-012-099-8763`

This is completely invalid. Phone Numbers are NXX-NXX-XXXX where N is between 2 and 9 and X is between 0 and 9 see [Wikipedia](https://en.wikipedia.org/wiki/North_American_Numbering_Plan) subsection Numbering system

**Comment by Marak** [comment]
Good catch.

Will accept a pull request to address this.

One thing to consider is Faker supports multiple localities, so there should be separate logic for generating phone numbers from different regions.

**Comment by greyhwndz** [comment]
Is it possible to generate random real city name given a country?

**Comment by evanjmg** [comment]
This is a huge issue. Chance has the same issue

**Comment by vedmant** [comment]
Yeah, it should have custom formats, currently I use just something like this '+4420' + _.shuffle([0,1,2,3,4,5,6,7,8,9]).slice(0, 8).join('')
